### PR TITLE
 new demo

### PR DESCRIPTION
**Pull request by makk94**

### CODE REVIEW

File: shell/demo/take1.sh [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    echo -e "-----------------------------------------------------------------------------------------------------------"
+    echo -e "[3] Running cleanup\n"
+    execute_cleanup $_temp
+    post_results $_test_id $_res
+    if ( echo "100 9 17 18 105 127" | grep -w -q $_res );then
+        results+=( "${GREEN}${names[$i]}\tPROTECTED${NC}" )
+    elif [ $_res -eq 101 ];then
+        results+=( "${RED}${names[$i]}\tUNPROTECTED${NC}" )
+    else
+        results+=( "${YELLOW}${names[$i]}\tERROR${NC}" )
```
Comment (privateducky) [inline shell/demo/take1.sh:120]: colors aren't working for me.. know why?

File: shell/demo/take1.sh [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+function check_relevance {
+    echo -e "${GREEN}[✓] Result: Success - server or workstation detected${NC}"
+}
+
+function download_test {
+    local _test_id=$1
+    local _temp=$2
+    location=$(curl -sfSL -w %{url_effective} -o $_temp -H "token:${PRELUDE_TOKEN}" -H "dos:${dos}" -H "id:${_test_id}" $PRELUDE_API)
```
Comment (privateducky) [inline shell/demo/take1.sh:49]: can we put things into the .vst directory, and clean that up at the end?

**Review by privateducky — CHANGES_REQUESTED** [review]
nice; a few minor modifications

File: shell/demo/take1.sh (hunk 1 above)
Comment (makk94) [reply shell/demo/take1.sh:120]: really? that's a shame. The internet told me `tput` is more universal/ better than escape sequences.. but if it's already not working on yours I can move back

File: shell/demo/take1.sh [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+declare -a names=(
+  'Health check'
+  'Will a long running VST be stopped properly?'
+  'Will your computer quarantine Royal Ransomware?'
+  'Will your computer quarantine a malicious Office document?'
+  'Will your computer quarantine Colour-Blind malware?'
+)
+declare -a results
+
+function get_info {
```
[the remark this answers is not in the document]
Comment (makk94) [reply shell/demo/take1.sh:34]: just print test name only?

[the remark this answers is not in the document]
Comment (privateducky) [reply shell/demo/take1.sh:34]: yep; ID and name is enough

**Review by privateducky — APPROVED** [review]

**Pull request merged**